src/steps/meta_commands/meta_command.ts

```diff
@@ -0,0 +1,31 @@
+export abstract class MetaCommand {
+    wrapperPath: string;
+    constructor(
+        public command: string | MetaCommand,
```

Comment:
```suggestion
        public command: Command,
```

I think, so it becomes self-referable, e.g. you can produce any chain you need.

Reply:
In this case something like new RetryCommand("yarn lint") is not allowed, no?

Reply:
fixed